### PR TITLE
feat: configuring logger level and release via environment variables,…

### LINKED ISSUE #132: A way to modify log level
I'd like to silence non-error messages in the console, e.g only have output from the actual `shellcheck` binary.

It looks like the `info` loglevel [is hardcoded here](https://github.com/gunar/shellcheck/blob/8f492ea4701c5960a134f14fbd73335a38113926/src/configs/config.ts#L105). Would you mind supporting a environment variable like `SHELLCHECK_NPM_LOG_LEVEL` so that users can control this log level?

### PR DESCRIPTION
**Pull request by carlocorradini**
Configuring logger level and release via environment variables
Deprecated node.js < 18.12.0

Fix #132

### CODE REVIEW

**Comment by silverwind** [comment]
Tested it by cloning the branch and building it, then `npm i -g .` and ran `SHELLCHECKJS_LOGGER_LEVEL=off shellcheck --help`, logs were suppressed as expected.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
     strategy:
       matrix:
         platform: [ubuntu-latest, macos-latest, windows-latest]
-        node-version: [16.x, 18.x]
+        node-version: [18.x, 20.x]
```
Comment (silverwind) [inline .github/workflows/ci.yml:44]:
```suggestion
        node-version: [18.x, 20.x, 22.x]
```

22 will go LTS in a few months, so good to test it already.

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
Need to run prettier :)

**Comment by carlocorradini** [comment]
@silverwind Done. I'll release this weekend

**Pull request merged**